src/vendedores.ts

```diff
@@ -1,31 +1,166 @@
-import { any } from "ramda";
+import { find, any, sum, props } from "ramda";
 
+// ⚡️⚡️Clase Vendedor
 abstract class Vendedor {
   abstract puedeTrabajarEn(ciudad: Ciudad): boolean;
+  abstract vendedorInfuyente(): boolean; // jm
+
+  vendedorVersatil(): boolean {
```

Comment:
Lo de `vendedor` dentro de `vendedorVersatil` se me hace un poco redundante, porque ya la clase se llama así. Es una práctica habitual ponerle prefijo `es` (o `is` en inglés) a los métodos que devuelven booleano... siguiendo eso lo llamaría `esFirme`.